src/components/DatePicker/index.ios.js

```diff
@@ -1,9 +1,8 @@
-import React from 'react';
-// eslint-disable-next-line no-restricted-imports
+import React, {useState, useRef, useCallback, useEffect} from 'react';
 import {Button, View, Keyboard} from 'react-native';
 import RNDatePicker from '@react-native-community/datetimepicker';
 import moment from 'moment';
-import _ from 'underscore';
+import isFunction from 'lodash/isFunction';
```

Comment:
I am curious about usage `lodash` instead of `underscore` at **Expensify**. I don't mind, but AFAIK we use only `get` from `lodash`. 
cc @fabioh8010 @gedu

Reply:
As I see we have various functions exported  from lodash (lodashFindLast, lodashMerge, lodashIntersection, lodashOrderBy, lodashUnset, lodashCloneDeep and others)

Reply:
@rezkiy37 Sorry for the late response, just noticed this one today. Yes, right now the usage of `lodash` / `underscore` is a bit confusing but we are going to fix this during the TS migration. We will remove `underscore` usage to only keep `lodash`, and also restrict `lodash` usage in favor of vanilla JS solutions when possible. [More info here](https://expensify.slack.com/archives/C01GTK53T8Q/p1687477811950459).